README.md

```diff
@@ -31,14 +31,12 @@ Generally Available
 
 Contact help@mapbox.com for information
 
-* [Distance](https://www.mapbox.com/developers/api/distance/)
-  * Travel-time tables between up to 100 points
+* [Matrix](https://www.mapbox.com/api-documentation/#matrix)
```

Comment:
I think this change is jumping the gun on #176 – once you revert this I think this PR will be good to go.